docs/04-reference/wingsdk-api.md

```diff
@@ -5514,6 +5515,16 @@ approxSize(): num
 
 Retrieve the approximate number of messages in the queue.
 
+##### `pop` <a name="pop" id="@winglang/sdk.cloud.IQueueClient.pop"></a>
+
+```wing
+pop(): str
```

Comment:
hmm, this is supposed to be `pop(): str?`, any clue what went wrong?

Reply:
Looks like a bug in the doc generation tool. I think it's related how we had to hack to support `Promise<T | undefined>` (something jsii doesn't allow at the moment). Can you open an issue?

Reply:
Done: https://github.com/winglang/wing/issues/2543